usr/src/uts/common/fs/zfs/zfs_rlock.c

```diff
@@ -92,124 +93,146 @@
  * growth all other writers and readers must be excluded.
```

Comment:
This comment should be updated to reflect that 128kb is no longer the maximum block size

Reply:
sure thing.